frontend/dockerfile/dockerfile_test.go

```diff
@@ -2666,6 +2667,7 @@ ENV foo=bar
 }
 
 func testExposeExpansion(t *testing.T, sb integration.Sandbox) {
+	integration.SkipIfDockerd(t, sb, "image export")
```

Comment:
as discussed image export could be handled using the `moby` one conditionally instead of skipping the test.